packages/sfp-cli/src/core/package/promote/PromoteUnlockedPackageImpl.ts

```diff
@@ -27,7 +26,7 @@ export default class PromoteUnlockedPackageImpl {
             result.id = packageVersionData.SubscriberPackageVersionId;
         } catch (e) {
             if (e.message.includes('previously released')) {
-                SFPLogger.log(`Package ${this.package_version_id} is already promoted, Ignoring`);
+                throw new Error(`The package version ${packageVersionData.MajorVersion}.${packageVersionData.MinorVersion}.${packageVersionData.PatchVersion} was already promoted in a previous build. For a given this version number, you can promote only one version.`);
```

Comment:
@dieffrei  I think we really need to check whats the error

Reply:
@azlam-abdulsalam what do you think if we add a new parameter, "--fail-when-is-promoted"?

Reply:
Yeah, it makes sense. Lets add it, and also ensure release command remains compatible

Reply:
@azlam-abdulsalam perhaps it was different in the past, but if you attempt to promote package which is already promoted the CLI does not throw an error. Instead it's a success and this message:
Successfully promoted the package version, ID: 04txxxxxxx